PowerEditor/src/NppIO.cpp

```diff
@@ -1934,7 +1935,7 @@ bool Notepad_plus::fileRename(BufferID id)
 			{
 				_nativeLangSpeaker.messageBox("RenameTabTemporaryNameIsEmpty",
 					_pPublicInterface->getHSelf(),
-					L"The specified name cannot be empty, or it cannot contain only space(s) or TAB(s).",
+					L"The specified name cannot be empty, or it cannot contain only space(s).",

```

Comment:
There's no reason to change it, since it's the fact.
Please keep as it was.

Reply:
Or alternatively maybe more generic "... cannot contain only white space(s)." ?